tests/conftest.py

```
@@ -163,7 +170,10 @@ def counting(root, tags, **kwargs):
             walks.append(len(tags))
         return original(root, tags, **kwargs)
 
-    monkeypatch.setattr(track_changes, "iter_revision_elements", counting)
+    for name, mod in list(sys.modules.items()):
+        if name == "docx_editor.track_changes" or name.startswith("docx_editor.track_changes."):
+            if getattr(mod, "iter_revision_elements", None) is original:
```

Ruff (0.16.2)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)